browser/components/shared/button/CloseButton.js

```diff
@@ -0,0 +1,30 @@
+import styled, { css } from "styled-components";
+import { Variants } from "../../../constants";
+import PropTypes from 'prop-types';
+import Icon from "@material-ui/core/Icon";
+
+function CloseButton(props) {
+    return (
+        <Root
+            onClick={props.onClick ?? props.onClick}
+            variant={props.variant}>
+            X
+        </Root>
+    );
+}
+CloseButton.propTypes = {
+    text: PropTypes.string,
+    onClick: PropTypes.func.isRequired
+}
+
+const Root = styled.button`
+    font-size: 8px;
+    height: 16px;
+    margin-top: 16px;
+    cursor: pointer;
+    border: 1px solid ${({ theme }) => theme.colors.dialogueTitle};
```

Comment:
Use `theme.colors.headings` instead and set the value in the theme file to "#ffffff"
Remove `dialogueTitle` from theme